.github/workflows/test-e2e.yml

```diff
@@ -10,29 +10,29 @@ on:
 
 jobs:
   # Install & build frontend
-  install:
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+  # install:
+  #   runs-on: ubuntu-22.04
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v3
 
-      - name: Cypress install
-        uses: cypress-io/github-action@v5
-        with:
-          # Disable running of tests within install job
-          runTests: false
-          build: npm run build
+  #     - name: Cypress install
+  #       uses: cypress-io/github-action@v5
+  #       with:
+  #         # Disable running of tests within install job
+  #         runTests: false
+  #         build: npm run build
```

Comment:
I completely disabled cacheing to start just so I could figure out how to get it to even boot and run against the dockerized app. I'm pleasantly surprised to see that this still runs the entire suite in ~11min. 

We could likely cut that down significantly with a proper build cache